youtube_dl/extractor/bbc.py

```diff
@@ -761,8 +762,13 @@ class BBCIE(BBCCoUkIE):
         'only_matching': True,
     }, {
         # custom redirection to www.bbc.com
+        # video with window.__INITIAL_DATA__
         'url': 'http://www.bbc.co.uk/news/science-environment-33661876',
-        'only_matching': True,
+        'id': 'p02xzws1',
+        'title': 'Pluto may have \'nitrogen glaciers\'',
+        'description': 'Pluto could have glaciers of nitrogen ice, new photographs from Nasa\'s New Horizons probe suggest.',
```

Comment:
Use outer double quotes instead of escaping single inner quotes.

Reply:
The escaped quotes were in the program output, but of course that's JSON and not Python ...

Reply:
The point is to always have copy-paste ready string literal in tests for debugging/testing purposes.